libpromises/failsafe.cf

```diff
@@ -69,6 +69,13 @@ bundle agent main
                     happen with the most up to date and complete
                     information.";
 
+      "Trigger Policy"
+        usebundle => failsafe_cfe_internal_trigger_policy,
```

Comment:
So you think it will be best to do this from embedded policy than to have it done from the C?

Reply:
I don't know yet. As I said, this is the first shot. But I think it makes sense to run `update.cf` and `promises.cf` in the same way.